src/components/layouts/ItemDetails.js

```diff
@@ -93,10 +94,38 @@ class PlaceDetails extends Component {
     getPlaceDetails = function() {
         const { detailsId, detailsType, setCurrentItem, fetchDetails } = this.props
 
-        let details = fetchDetails(detailsId, detailsType)
+        fetchDetails(detailsId, detailsType).then(details => {
+            console.log('Place details from fetchDetails: ', details)
+
+            this.getRecommendations(details.properties.vibes)
+        })   
         
     }
 
+    // Very basic content-based filtering recommendation 
+    getRecommendations = function (vibes) {
```

Comment:
If I'm understanding this correctly, getRecommendations returns the top 7 values from getPicks. getPicks essentially passes a query of places within a certain search radius, or other filters like time of day eventually, into the scorePlaces function, which chugs out absolute scores for all the places, based upon the factors we include in the scoreBy list?

Reply:
If that is the case, then eventually, for instances where we decide to do so, the recommender system should be replacing the scorePlaces function right? And instead of using absolute scores with just arithmetic, we use user profiles, similarity scoring, ML, etc. but we would want the output format to be the same as scorePlaces, so we can insert the recsys pretty easily